apps/automation/automation-v2/src/tasks/spice-auction-burn-and-notify.ts

```
@@ -7,28 +7,30 @@ import { chainFromId, getMainnetSubmissionParams } from "@/config";
 // avoid hot spots in code when iterating
 const MAX_BACKCHECK_LENGTH = 10;
 
 type EpochInfo = 
-  AbiParametersToPrimitiveTypes<ExtractAbiFunction<typeof AuctionBase.ABI, "getEpochInfo">["outputs"]>[0];
+  AbiParametersToPrimitiveTypes<ExtractAbiFunction<typeof Spice.ABI, "getEpochInfo">["outputs"]>[0];
 
 type SendParamOptions =
   AbiParametersToPrimitiveTypes<ExtractAbiFunction<typeof TempleGold.ABI, "quoteSend">["inputs"]>[0];
 
+type Auction = GetContractReturnType<typeof Spice.ABI, PublicClient>;
+type TempleGold = GetContractReturnType<typeof TempleGold.ABI, PublicClient>;
```

Biome (2.1.2)

[error] Shouldn't redeclare 'TempleGold'. Consider to delete it or rename it.

'TempleGold' is defined here:


(lint/suspicious/noRedeclare)